lib/gitarro/backend.rb

```diff
@@ -244,19 +244,32 @@ def reviewed_pr?(comm_st, pr)
 
   # this function will check if the PR contains in comment the magic word
   # # for retrigger all the tests.
+  # TODO: To be deprecated when retriggered_by_checkbox? is welcomed by users
```

Comment:
here 2 remarks:

1) TODO in code are will get lost and forgotten :smile: . So we don't want this here, but in a GitHub issue is better.

2) and more important: the usage of `gitarro` by SUSE-Manager doesn't dictate the direction of `gitarro` itself, which is a generic opensource project.
In this particolar case, if in the SUSE-Manager/Uyuni project you don't use the comment, this is a particular use-case, no need to deprecate the comment feature.

In any case the `comment` feature , is needed for retrigger failed test, so we cannot deprecate it, since they are 2 diff features.

Reply:
> "In any case the comment feature , is needed for retrigger failed test"

No need anymore with my PR, that's the idea ;)

Reply:
I don't think the checkbox should remove the comment. The comment workflow is more universal. kuberentes and other communities use this workflow.

Reply:
Nono I don't think we should remove it. Let's keep both, yes!
What I mean is that in Spacewalk, we will use the checkbox for re-trigger failure tests

Reply:
ok thx for clarification :) imho in spacewalk an user could choose to use both.
:sunflower: